doc/RELEASE_WALKTHROUGH.rst.txt

```diff
@@ -90,7 +92,7 @@ Build wheels
 ------------
 
 Trigger the wheels build by pointing the numpy-wheels repository at this
-commit. This can take a while. The numpy-wheels repository is cloned from
```

Comment:
Might put in an estimate. My rough guess is 1/2 hour for the azure builds.

Reply:
ok